homeassistant/components/apple_tv/__init__.py

```diff
@@ -50,16 +57,27 @@ def ensure_list(value: Union[T, Sequence[T]]) -> Sequence[T]:
     return value if isinstance(value, list) else [value]
 
 
+CREDENTIALS_SCHEMA = vol.Schema(
+    {
+        vol.Optional(CONF_CREDENTIALS_MRP): cv.string,
+        vol.Optional(CONF_CREDENTIALS_DMAP): cv.string,
+        vol.Optional(CONF_CREDENTIALS_AIRPLAY): cv.string,
+    }
+)
+
 CONFIG_SCHEMA = vol.Schema(
     {
         DOMAIN: vol.All(
             ensure_list,
             [
                 vol.Schema(
                     {
-                        vol.Required(CONF_HOST): cv.string,
-                        vol.Required(CONF_LOGIN_ID): cv.string,
-                        vol.Optional(CONF_CREDENTIALS): cv.string,
+                        vol.Required(CONF_ADDRESS): cv.string,
+                        vol.Required(CONF_IDENTIFIER): cv.string,
+                        vol.Required(CONF_PROTOCOL): vol.In(
+                            [PROTOCOL_DMAP, PROTOCOL_MRP]
```

Comment:
I think you don't need to add this config since you can get the info from the credentials map.